www/front_src/src/Resources/Actions/Resource/Downtime/Dialog.tsx

```diff
@@ -94,6 +97,14 @@ const DialogDowntime = ({
     setFieldValue(field, value);
   };
 
+  React.useEffect(() => {
+    moment.locale(locale);
+  }, [locale]);
+
+  React.useEffect(() => {
+    moment.tz.setDefault(timezone);
+  }, [timezone]);
+
```

Comment:
I think these effects are a bit superfluous. I think we should load the dialog until all data needed are available (including locale and timezone).

Reply:
I could put it in a single useEffect with "loading" as dependency, but I don't think it's the best way.
So I think I don't understand what is exactly your point ?

Reply:
No that's not what I mean. I mean that in the Dialog, locale and timezone are not supposed to be null, so we should have only one hook call setting them at the beginning. Also the Dialog is not supposed to be usable if they're not available yet

Reply:
we'll probably refactor this once user information are stored in global context